snap/squashfs/squashfs.go

```diff
@@ -403,23 +412,29 @@ func (s *Snap) Build(sourceDir, snapType string, excludeFiles ...string) error {
 	if err != nil {
 		return err
 	}
+	// default to xz
+	compression := opts.Compression
+	if compression == "" {
+		compression = "xz"
```

Comment:
Could we add a short note about the implications of `xz` from @anonymouse64's forum post?

Reply:
I think something like this would be good:

```
// TODO: support other compression options, xz is very slow for certain apps, see https://forum.snapcraft.io/t/squashfs-performance-effect-on-snap-startup-time/13920
```

Reply:
Added